db/schema.sql

```diff
@@ -8,20 +8,20 @@ CREATE TABLE version
 );
 
 /* Set the current database version */
-INSERT INTO version VALUES (0);
+INSERT INTO version VALUES (1);
```

Comment:
Is there some way you can check this? So noone foregets to update it. I'm also guessing it has to be synced with `DATABASE_VERSION` in `config/default.py`.

Reply:
Yes, it should be pretty easy to add a test case for it. This file reflects the current database if it was created from scratch, so only updating the config is definitely an error, even though it will not affect the already running database.